src/StatusPageEmbed.js

```diff
@@ -272,7 +272,7 @@ class StatusPageEmbed extends Component {
           ) }
         </div>
         <div className="StatusPageEmbed__close">
-          <button className="StatusPageEmbed__close__button" onClick={() => this.dismiss()}>
+          <button className="StatusPageEmbed__close__button" aria-label="Close" onClick={() => this.dismiss()}>
             <FontAwesomeIcon icon={faTimes} />
```

Comment:
do we `aria-hidden` this already?

Reply:
Don't think so

Reply:
https://www.456bereastreet.com/archive/201609/hiding_inline_svg_icons_from_screen_readers/ suggests we should, but I suspect `FontAwesomeIcon` is not ours..

Reply:
![Screenshot_2020-02-14 Font Awesome How to Use](https://user-images.githubusercontent.com/2552726/74536859-d2b01100-4f30-11ea-990f-7e210b508cbd.png)

Reply:
Yeah it's not...

Reply:
There you go...